src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleFileFunction.java

```diff
@@ -532,6 +543,45 @@ private static ModuleThreadContext execModuleFile(
     return context;
   }
 
+  // Adds a local_repository for each repository injected via --injected_repositories.
+  private static void injectRepos(
+      Map<String, PathFragment> injectedRepositories,
+      ModuleThreadContext context,
+      StarlarkThread thread)
+      throws EvalException {
+    // Use the innate extension backing use_repo_rule.
+    ModuleThreadContext.ModuleExtensionUsageBuilder usageBuilder =
+        new ModuleThreadContext.ModuleExtensionUsageBuilder(
+            context,
+            "//:MODULE.bazel",
+            "@bazel_tools//tools/build_defs/repo:local.bzl%local_repository",
+            /* isolate= */ false);
+    ModuleFileGlobals.ModuleExtensionProxy extensionProxy =
+        new ModuleFileGlobals.ModuleExtensionProxy(
+            usageBuilder,
+            ModuleExtensionUsage.Proxy.builder()
+                .setDevDependency(true)
```

Comment:
does this matter at all? just checking. If not, I have a slight preference for not setting it for least surprise, but not a big deal.

Reply:
The value doesn't matter as far as I can tell, but we need to set one or the builder throws. `true` seems more fitting to me than `false` since `--inject_repository` repos won't be available to consumers of the root module.